src/app/app.component.ts

```diff
@@ -75,6 +76,17 @@ export class AppComponent implements OnInit, OnDestroy {
                 });
             }
         );
+
+        const newNotificationSubscription = this.notificationService.newNotification$.subscribe(
+            () => {
+                this.notificationBlink = 'rgba(255, 255, 255, 0.5)';
+                setTimeout(() => {
```

Comment:
Hmm, `setTimeout` is a little bit ugly, and can give issues later on with zones. Any chance of replacing this with a proper Observables approach?

Reply:
You mean by a `delay` operator for example, right? That should be easy to change. 
We use `setTimeout` also for the polling mechanisms. Would you replace it everywhere?

Reply:
Alright. Thank you :)